apps/cms-v2/src/pages/api/webhooks/product-updated.ts

```diff
@@ -58,23 +60,31 @@ const handler: NextWebhookApiHandler<ProductUpdatedWebhookPayloadFragment> = asy
   const { authData, payload } = context;
 
   if (!payload.product) {
+    logger.warn("Product not found in payload");
     Sentry.captureException("Product not found in payload");
 
     return res.status(500).end();
   }
 
-  logger.info("Webhook called");
+  logger.info("Webhook called", {
+    productId: payload.product.id,
+    variantsLength: payload.product.variants?.length,
+    productName: payload.product.name,
+    channelsIds: payload.product.channelListings?.map((c) => c.channel.id) || [],
+  });
```

Comment:
question: Do you think logging such info once at the beginning of the call is enough? I found this solution as good for remove duplication but when couple wehbooks were fired I didn't know which logs is for which webhook call.

Reply:
We are solving observability problems here, not duplication.

If you can query in datadog log with other attribute, like `"Webhook called" AND @path="api/webhooks/product-updated" `, it may be fine. 

But you can also create some namespace like `[PRODUCT_UPDATED] Webhook called` etc. In the end you need this to be easily searchable by message and attributes